constants.go

```diff
@@ -467,6 +467,8 @@ const (
 	// CertExtensionScopePin is used to pin a certificate to a specific scope and
 	// set of scoped roles.
 	CertExtensionScopePin = "scope-pin@goteleport.com"
+	// CertExtensionAgentScope is used to pin a host certificate to a specific scope.
+	CertExtensionAgentScope = "agent-scope@goteleport.com"
```

Comment:
IIUC `scope_pin` is intended for _users_ and `agent_scope` is meant for Teleport instances. Could we better clarify in the comments the difference between these two?